crates/revmc-worker/src/worker/aot.rs

```diff
@@ -17,7 +17,7 @@ impl AotRuntime {
 
     pub(crate) fn compile(
         &self,
-        name: &'static str,
+        code_hash: FixedBytes<32>,
         bytecode: &[u8],
         spec_id: SpecId,
```

Comment:
make type defined in revm primitives more readable 
```suggestion
        &self,
        code_hash: B256,
        bytecode: Bytes,
        spec_id: SpecId

```

Reply:
done